source/_integrations/timer.markdown

```diff
@@ -25,6 +25,9 @@ To add a timer to your installation, add the following to your `configuration.ya
 timer:
   laundry:
     duration: "00:01:00"
+
+  pool:
+    duration: "01:00:00"
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.